tdrs-backend/tdpservice/search_indexes/models.py

```diff
@@ -0,0 +1,271 @@
+"""Models representing parsed data file records submitted to TDP."""
+
+from django.db import models
+
+# Create your models here.
+
+
+class ParsedDataRecord(models.Model):
+    """Inheritable Django Model with fields shared by subtypes."""
+
+    record = models.CharField(max_length=156, null=False, blank=False)
```

Comment:
@jtimpe 156 is the max len for T1 and T3 records. but each individual element has a specific expected len as shown here: https://www.acf.hhs.gov/sites/default/files/documents/ofa/tanf_data_report_section1_10_2008.pdf 

for references, all record layouts for T1-T7 (for TANF and Tribal TANF) and M1-M7 (for SSP-MOE) here: https://www.acf.hhs.gov/ofa/policy-guidance/final-tanf-ssp-moe-data-reporting-system-transmission-files-layouts-and-edits#tanf-layouts